gidgethub/sansio.py

```diff
@@ -369,7 +369,7 @@ def decipher_response(
 DOMAIN = "https://api.github.com"
 
 
-def format_url(url: str, url_vars: Mapping[str, Any]) -> str:
+def format_url(url: str, url_vars: Mapping[str, Any], *, base_url: str = DOMAIN) -> str:
```

Comment:
Isn't it an antipattern to have mutable arg defaults?

Reply:
What mutable default argument? (And it isn't a black-and-white anti-pattern; you just have to be careful.)